src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,170 @@
+.wrapper {
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+  border-radius: 50%;
+  border: 1px solid rgba($color: #848080, $alpha: 0.4);
+  background: $stopwatch-container-background;
+  box-shadow: $stopwatch-container-shadow;
+  padding: 20px;
+
+  &__face {
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 100%;
+    transform: rotate(90deg);
+    border-radius: 50%;
+    background: $stopwatch-face-background;
+    box-shadow: $stopwatch-face-shadow;
+  }
+
+  &__markup {
+    width: 100%;
+    height: 100%;
+    border-radius: 50%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    &-block {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      position: absolute;
+      width: 100%;
+      height: 50px;
+      padding: 0 15px;
+
+      @include rotateMarkupBlocks();
+    }
+
+    &-item {
+      height: 0.5rem;
+      width: 0.5rem;
+      background: $stopwatch-markup-color;
+      border-radius: 50%;
+
+      @media screen and (max-width: 500px) {
+        height: 0.25rem;
+        width: 0.25rem;
+      }
+    }
+  }
+
+  &__arrows {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    width: 100%;
+    height: 100%;
+    border-radius: 50%;
+
+    &::before {
+      content: "";
+      position: absolute;
+      display: inline-block;
+      width: 1.5rem;
+      height: 1.5rem;
+      border-radius: 50%;
+      top: 50%;
+      left: 50%;
+      transform: translate(-50%, -50%);
+      background: $stopwatch-minute-arrow-color;
+      box-shadow: 1px 1px 3px rgba(24, 30, 36, 0.65);
+      z-index: 1;
+    }
+
+    &::after {
+      content: "";
+      position: absolute;
+      width: 0.8rem;
+      height: 0.8rem;
+      background: #fd382d;
+      box-shadow: 1px 1px 3px rgba(24, 30, 36, 0.65);
+      border-radius: 50%;
+      top: 50%;
+      left: 50%;
+      transform: translate(-50%, -50%);
+      z-index: 5;
+    }
+  }
+
+  &__arrow {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+
+    &--minute {
+      width: 67.5%;
+      height: 67.5%;
+
+      animation: arrowMove 60s linear infinite;
+
+      @media screen and (max-width: 768px) {
+        height: 60%;
+        width: 60%;
+      }
+
+      &::after {
+        content: "";
+        position: absolute;
```

Comment:
Don't repeat yourself: if your use the same styles for the both classes with modificator, move this dtyles to the general class